Improving step-by-step guide for vulnerability detector (#7149)
wazuh/wazuh-documentation: source/_templates/installations/manager/configure_indexer_connection.rst

.. Copyright (C) 2015, Wazuh, Inc.

> By default, the host is set to localhost: ``<host>https://0.0.0.0:9200</host>``. Replace ``0.0.0.0`` with your Wazuh indexer node IP address or hostname.

Author: Done!

> By default, the host is set to localhost: ``<host>https://0.0.0.0:9200/</host>``.  Replace ``0.0.0.0`` with your Wazuh indexer node IP address or hostname

Reviewer: ```suggestion
By default, the host is set to localhost: ``<host>https://0.0.0.0:9200/</host>``.  Replace ``0.0.0.0`` with your Wazuh indexer node IP address or hostname.
```

> By default, the host is set to localhost: ``<host>https://0.0.0.0:9200/</host>``. Replace ``0.0.0.0`` with your Wazuh indexer node IP address or hostname.

Reviewer: ```suggestion
By default, the host is set to localhost: ``<host>https://0.0.0.0:9200</host>``. Replace ``0.0.0.0`` with your Wazuh indexer node IP address or hostname.
```

> .. code-block:: xml

Reviewer 2: Check alignment. We use three spaces indentation and we should check the configuration block sample alignment as well. Maybe something like this

```
.. code-block:: xml
   :emphasize-lines: 4

   <indexer>
     <enabled>yes</enabled>
     <hosts>
       <host>https://0.0.0.0:9200</host>
     </hosts>
     <ssl>
       <certificate_authorities>
         <ca>/etc/filebeat/certs/root-ca.pem</ca>
       </certificate_authorities>
       <certificate>/etc/filebeat/certs/filebeat.pem</certificate>
       <key>/etc/filebeat/certs/filebeat-key.pem</key>
     </ssl>
   </indexer>

```

Author: Done!

Reviewer 2: Sorry, I wasn't clear. I mean the alignment of Sphynx code only, not code inside code blocks. For example, we use

```
.. code-block:: xml
   :emphasize-lines: 4

   some code here
```

rather than, say

```
.. code-block:: xml
     :emphasize-lines: 4

 some code here

```

> By default, the host is set to localhost: ``<host>https://0.0.0.0:9200/host>``. Replace it with your Wazuh indexer address accordingly. You can use either IP addresses or hostnames.

Reviewer 3: Typo: `9200/host>`
Missing `<`

---

I think that we should keep the 0.0.0.0 IP if it is the default value but mention the change, something like:

> Change 0.0.0.0 by your Wazuh indexer node IP address or hostname.

But only for this section, the following code blocks can maintain the custom IPs as they seem to be an example, what do you think?

Reviewer: ```suggestion
By default, the host is set to localhost: ``<host>https://0.0.0.0:9200</host>``. Replace it with your Wazuh indexer address accordingly. You can use either IP addresses or hostnames.
```

  :emphasize-lines: 4

  <indexer>
      <enabled>yes</enabled>
      <hosts>

>         <host>https://10.0.0.1:9200</host>

Reviewer: ```suggestion
        <host>https://0.0.0.0:9200</host>
```

      </ssl>
  </indexer>

> If you have more than one Wazuh indexer node, add all the required ``<host>`` entries. The vulnerability detector module will report to the first available node found and will switch to the next one in case of error.

Reviewer 2: ```suggestion
If you have a Wazuh indexer cluster, add a ``<host>`` entry for each one of your nodes. For example, in a two-nodes configuration:
```

Author: Done!

> .. code-block:: xml

Reviewer 2: Check alignment

```
.. code-block:: xml

   <hosts>
     <host>https://10.0.0.1:9200</host>
     <host>https://10.0.0.2:9200</host>
   </hosts>
```

Author: Fixed!

Reviewer 2: Same comment here

      <hosts>
        <host>https://10.0.0.1:9200</host>
        <host>https://10.0.0.2:9200</host>
      </hosts>

> 

Reviewer 2: ```suggestion
Vulnerability detection prioritizes reporting to the first node in the list. It switches to the next node in case it's not available.

```

Author: Added!
